paasta_tools/api/views/instance.py

```diff
@@ -268,7 +302,7 @@ def instance_status(request):
         error_message = traceback.format_exc()
         raise ApiFailure(error_message, 500)
 
-    if instance_type != 'flinkcluster':
+    if instance_type != 'flinkcluster' and instance_type != 'tron':
```

Comment:
Why do we need this?

Reply:
version = get_deployment_version(actual_deployments, settings.cluster, instance) returns None.